CHANGES.rst

```diff
@@ -5,6 +5,11 @@ Changes in Npgsql-Plugin for Crate
 Unreleased
 ==========
 
+2019/08/24 1.2.4
+================
+
+- Breaking change: Changed mapping for geo_point to npgsql point datatype (oid 600)
```

Comment:
Maybe we could clarify that this change only has an effect if the CrateDB nodes are >= 4.1

```suggestion
-  Added a type mapping from ``geo_point`` (oid 600) to ``NpgsqlPoint``, to be
   able to handle ``geo_point`` values with CrateDB >= 4.1. There the streaming
   changed, so that ``geo_point`` values are no longer streamed as double array
   but as ``geo_point``.
```

Reply:
ok thx thats sounds good. One more question.. should the new version of the npgsql-crate plugin support crate version >= 4.1 and crate version <4.1? I did a small change to support both, but before i commit it can you please give me your opinion. thx

Reply:
Yes ideally it would support both